crates/jstz_node/src/main.rs

```diff
@@ -1,26 +1,21 @@
 use std::io;
 
-use actix_web::{middleware::Logger, web::Data, App, HttpServer};
 use clap::Parser;
-use env_logger::Env;
-use octez::OctezRollupClient;
-use services::{LogsService, Service};
-use tokio_util::sync::CancellationToken;
 
-use crate::services::{AccountsService, OperationsService};
+use crate::{
+    config::{
+        DEFAULT_KERNEL_FILE_PATH, DEFAULT_ROLLUP_NODE_RPC_ADDR, DEFAULT_ROLLUP_RPC_PORT,
+    },
+    node_runner::run_node,
+};
 pub use error::{Error, Result};
 
+mod config;
 mod error;
+mod node_runner;
 mod services;
 mod tailed_file;
```

Comment:
I think these can be dropped and you simply import them from the crate defined in `jstz_node` (introducing `lib.rs` now defines a crate + a binary in `jstz_node`)

Reply:
we need both crate and binary since we can still run just jstz-node, or run it from the sandbox